app/javascript/packs/admin_invoices/content.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function timeToDecimal(time) {
+  const arr = time.split(':');
+  return parseFloat(parseInt(arr[0], 10) + '.' + parseInt((arr[1]/6)*10, 10));
+}
+
+function getBusinessDatesCount(startDate, endDate) {
+  let count = 0;
+  const curDate = startDate;
+  while (curDate <= endDate) {
+    const dayOfWeek = curDate.getDay();
+    if (!((dayOfWeek === 6) || (dayOfWeek === 0))) {
+      count += 1;
+    }
+    curDate.setDate(curDate.getDate() + 1);
+  }
+  return count;
+}
+
+class Content extends React.Component {
+  render() {
+    const timelogs = this.props.invoices.sort((a, b) => (
+      a.start_time > b.start_time
+    ));
+    const totalDuration =
```

Comment:
this is not bad or good.
Another approach would be to prepare most unchangable data on back end, just not to play with Math and calculating on front end.